### DISCUSSION TITLE
Documentation: markdown style convention

### DISCUSSION THREAD

**Discussion by tai271828** [solvcon/devenv#60] [General (discussions)]
There is no standard but only suggested markdown style convention. Maybe we could get some consensus when writing/updating our document to prevent communication overhead like this https://github.com/solvcon/devenv/pull/50#pullrequestreview-704027489

Two conventions come to my mind at the time of writing. They are all syntactically legal :
1. One line break right after a head, sub-head ...etc.
2. Two line breaks at the end of a session, equation, and code block. I did not see many people adapt this convention but I personally like it for friendly reading.

If you know any best practice, please share with the team. Thanks!

**Comment by yungyuc** [comment]
> There is no standard but only suggested markdown style convention. Maybe we could get some consensus when writing/updating our document to prevent communication overhead like this [#50 (review)](https://github.com/solvcon/devenv/pull/50#pullrequestreview-704027489)
> 
> Two conventions come to my mind at the time of writing. They are all syntactically legal :
> 
> 1. One line break right after a head, sub-head ...etc.

I'd like to go with this style.

> 2. Two line breaks at the end of a session, equation, and code block. I did not see many people adapt this convention but I personally like it for friendly reading.

I do not see many use this convention, and I do not see it looks nice.

> 
> If you know any best practice, please share with the team. Thanks!
